strawberry/field.py

```diff
@@ -208,7 +227,9 @@ def _python_name(self) -> Optional[str]:
     def _set_python_name(self, name: str) -> None:
         self.name = name
 
-    python_name: str = property(_python_name, _set_python_name)  # type: ignore[assignment]  # noqa: E501
+    python_name: str = property(  # type: ignore[assignment]
+        _python_name, _set_python_name
+    )
```

Comment:
Since you touched this code, is there a reason to why it is like this and not using `@property`/`@python_name.setter`?

Reply:
This was just `pre-commit` formatting, so I'm not sure